src/test/java/bio/terra/service/filedata/DrsServiceTest.java

```diff
@@ -102,8 +142,8 @@ public void before() throws Exception {
   public void testLookupPositive() {
     DRSObject drsObject = drsService.lookupObjectByDrsId(authUser, drsObjectId, false);
     assertThat(drsObject.getId(), is(drsObjectId));
-    assertThat(drsObject.getSize(), is(fsFile.getSize()));
-    assertThat(drsObject.getName(), is(fsFile.getPath()));
+    assertThat(drsObject.getSize(), is(googleFsFile.getSize()));
+    assertThat(drsObject.getName(), is(googleFsFile.getPath()));
```

Comment:
The size and path are the same as the `azureFsFile` Maybe make those different to ensure that you're still getting the right one?